en/routing.texy

```diff
@@ -394,6 +394,25 @@ Flags can be freely combined using bitwise OR operator (`|`) and it is possible
 Route::$defaultFlags |= Route::CASE_SENSITIVE;
 \--
 
+.[caution]
+Route::$defaultFlags at Nette 2.4 is deprecated
+
+Substitude for `Route::$defaultFlags`
+---------------------------------
+
+If we use HTTP or HTTPS only, so the longer have to worry a tipping application. Of course, if we use HTTP and HTTPS cuncurrently, and we need another for subdomain (for example), than we can do it by easy write to file in main directory of application .htaccess
```

Comment:
If we use only HTTP or only HTTPS, we no longer have to worry about redirecting. If we need to use HTTP and HTTPS concurrently, we can do it in `.htaccess` file in the main directory of your application: